src/utils/flagCodes.ts

```diff
@@ -8,7 +8,7 @@ export interface Flag<T extends string> {
 
 // OUR specific flags
 // lowercase hex value to be appended to tx.data
-const SENTINEL = 'dec0de'
+const SENTINEL = CONFIG.transactions.sentinel
```

Comment:
We should add raise an error if:
* It's not a hex
* If it's a bad hex?

We should define what is a good code, and verify it here